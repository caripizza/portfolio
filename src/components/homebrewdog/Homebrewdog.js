import React from 'react';
import homebrewPic1 from '../../../assets/Homebrewdog/search-beers_homebrewdog.png';
import homebrewPic2 from '../../../assets/Homebrewdog/beer-search-results_homebrewdog.png';
import homebrewPic3 from '../../../assets/Homebrewdog/beer-detail_homebrewdog.png';
import homebrewPic4 from '../../../assets/Homebrewdog/profile_homebrewdog.png';

const Homebrewdog = () => {
  const imgStyle = {
    width: '50vw',
    marginBottom: '3em',
    border: '2px solid cyan'
  };

  const pStyle = {
    width: '70vw',
    background: 'gold',
    padding: '1em'
  };

  const hStyle = {
    padding: '.5em 2em 0 .7em',
    fontStyle: 'italic',
    fontWeight: 300
  };

  return (
    <>

      <main>
        <section style={{
          display: 'flex',
          flexDirection: 'column',
          alignItems: 'center',
          textAlign: 'center',
          backgroundImage: 'repeating-linear-gradient(orange, pink)',
          backgroundRepeat: 'repeat',
          backgroundPosition: 'center'
        }}>
          <h1 style={hStyle}>Homebrewdog</h1>
          <p style={pStyle}>Featuring a pop-up search modal, allowing you to search for beers by key word:</p>
          <img src={homebrewPic1} style={imgStyle}/>
          <p style={pStyle}>Yielding a list of search results, based on your beer query:</p>
          <img src={homebrewPic2} style={imgStyle}/>
          <p style={pStyle}>Clicking on a beer from the list directs to the beer detail, where you can rate and save the beer to your Profile:</p>
          <img src={homebrewPic3} style={imgStyle}/>
          <p style={pStyle}>Your profile displays a list of saved beers, allowing you to post and update comments for each, and even view aggregations based on IBU and ABV:</p>
          <img src={homebrewPic4} style={imgStyle}/>
        </section>
      </main>

    </>
  );
};

export default Homebrewdog;
