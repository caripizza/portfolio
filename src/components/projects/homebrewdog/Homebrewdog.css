.Homebrewdog {

  @media screen and (min-width: 600px) {
    img:not([alt="github"]) {
      width: 30em;
    }
  }

}
