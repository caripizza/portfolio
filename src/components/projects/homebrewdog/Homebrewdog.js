import React from 'react';
import { Link } from 'react-router-dom';
import styles from './Homebrewdog.css';
import homebrewPic1 from '../../../../assets/Homebrewdog/search-beers_homebrewdog.png';
import homebrewPic2 from '../../../../assets/Homebrewdog/beer-search-results_homebrewdog.png';
import homebrewPic3 from '../../../../assets/Homebrewdog/beer-detail_homebrewdog.png';
import homebrewPic4 from '../../../../assets/Homebrewdog/profile_homebrewdog.png';
import github from '../../../../assets/github.png';

const Homebrewdog = () => {
  return (
    <main className={styles.Homebrewdog}>
      <section>
        <h1>Homebrewdog</h1>
        <span>
          <a href="https://homebrewdog.herokuapp.com"
            target="_blank" rel="noopener noreferrer"
          >
              https://homebrewdog.herokuapp.com
          </a>
        </span>
        <br/>
        <a href="https://github.com/team-thermometer"
          target="_blank" rel="noopener noreferrer"
        >
          <img src={github} alt="github" className={styles.github}/>
        </a>
        <p>Featuring a pop-up search modal, allowing you to search for beers by key word:</p>
        <img src={homebrewPic1} alt="Homebrewdog-pic1" className={styles.screenshots}/>
        <p>Yielding a list of search results, based on your beer query:</p>
        <img src={homebrewPic2} alt="Homebrewdog-pic2" className={styles.screenshots}/>
        <p>Clicking on a beer from the list directs to the beer detail, where you can rate and save the beer to your Profile:</p>
        <img src={homebrewPic3} alt="Homebrewdog-pic3" className={styles.screenshots}/>
        <p>Your profile displays a list of saved beers, allowing you to post and update comments for each, and even view aggregations based on IBU and ABV:</p>
        <img src={homebrewPic4} alt="Homebrewdog-pic4" className={styles.screenshots}/>

        <Link to="/projects">Back</Link>
        <br/>

      </section>
    </main>
  );
};

export default Homebrewdog;
