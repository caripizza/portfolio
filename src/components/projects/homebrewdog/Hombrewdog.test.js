import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Homebrewdog from './Homebrewdog';

describe('Homebrewdog', () => {
  it('matches a snapshot', () => {
    const component = renderer.create(
      <MemoryRouter>
        <Homebrewdog />
      </MemoryRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
