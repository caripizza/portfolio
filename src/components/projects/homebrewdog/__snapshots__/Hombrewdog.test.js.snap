// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homebrewdog matches a snapshot 1`] = `
<main
  className="Homebrewdog"
>
  <section>
    <h1>
      Homebrewdog
    </h1>
    <span>
      <a
        href="https://homebrewdog.herokuapp.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        https://homebrewdog.herokuapp.com
      </a>
    </span>
    <br />
    <p>
      Featuring a pop-up search modal, allowing you to search for beers by key word:
    </p>
    <img
      alt="Homebrewdog-pic1"
      src=""
    />
    <p>
      Yielding a list of search results, based on your beer query:
    </p>
    <img
      alt="Homebrewdog-pic2"
      src=""
    />
    <p>
      Clicking on a beer from the list directs to the beer detail, where you can rate and save the beer to your Profile:
    </p>
    <img
      alt="Homebrewdog-pic3"
      src=""
    />
    <p>
      Your profile displays a list of saved beers, allowing you to post and update comments for each, and even view aggregations based on IBU and ABV:
    </p>
    <img
      alt="Homebrewdog-pic4"
      src=""
    />
    <a
      href="/projects"
      onClick={[Function]}
    >
      Back
    </a>
    <br />
  </section>
</main>
`;
